server/controllers/finance/reports/invoices/receipt.xlsx.js

```diff
@@ -0,0 +1,194 @@
+// Require library
+const xl = require('excel4node');
```

Comment:
This is pretty cool!

Could you write a small unit test to show that this works?  That way we can be sure changes in the future do not break this renderer.